app/services/organization_service.rb

```diff
@@ -0,0 +1,12 @@
+class OrganizationService
+  def save_organization_and_cities_organization(organization: Organization.new,
+                                                 city_ids: [])
+    return false unless organization.valid?
+
+    organization.save
+
+    city_ids.each do |city_id|
+      CitiesOrganization.create(city_id: city_id, organization_id: organization.id)
```

Comment:
You should not have to create an intermediate records - Rails will take care of that. Take a closer look at `physician.patients = patients` https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association